Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gray_matter"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
https://github.com/the-alchemists-of-arland/gray-matter-rs/blob/aa5b0b42c32ed7d35e628d5d72acfe343c14c3b3/.github/workflows/publish.yml#L38-L46

In fact, you don't need to set the version. Just leave it with `version = "0.1.0"`. This fake version will replaced by triggering github action workflow `publish.yaml` once a tag is released.

Reply:
Cool! That being said, I think having the correct version reflected in the `Cargo.toml` file is the correct approach nontheless, so one can build the crate manually from the repo. I'll leave it in